### PR TITLE
Fix Exporter Carp dependency: s/1.05/1.50/

### PR DESCRIPTION
**Pull request by lalten**
This was probably a typo? There is no Carp 1.05.

### CODE REVIEW

**Comment by mauke** [comment]
There is a Carp 1.05: 8c3d97219390037dbdf177595955b394d8b3290e

**Review by haarg — DISMISSED** [review]

File: dist/Exporter/Makefile.PL [hunk 1]
```diff
@@ -9,7 +9,7 @@ WriteMakefile(
     VERSION_FROM => 'lib/Exporter.pm',
     ( $] > 5.011 ) ? () : ( INSTALLDIRS => 'perl' ),    # CPAN sourced versions should now install to site
     PREREQ_PM => {
-        'Carp' => '1.05',
+        'Carp' => '1.50',
```
Comment (mauke) [inline dist/Exporter/Makefile.PL:12]: Carp 1.50 was first released with perl 5.28 (see [perl5280delta](https://perldoc.perl.org/perl5280delta#Updated-Modules-and-Pragmata)). This change would mean that Exporter now requires perl 5.28+ when previously it worked fine on 5.8. I don't think that's acceptable.

**Review by mauke — CHANGES_REQUESTED** [review]

File: dist/Exporter/Makefile.PL (hunk 1 above)
Comment (Grinnz) [reply dist/Exporter/Makefile.PL:12]: Carp is dual-life. That said, there is no reason for this change.

**Comment by tonycoz** [comment]
Yes, the prereq is correct, closing.

**Pull request closed without merging**